tools/pipelines/templates/build-npm-client-package.yml

```
@@ -727,6 +742,12 @@ extends:
                   lfs: '${{ parameters.checkoutSubmodules }}'
                   submodules: '${{ parameters.checkoutSubmodules }}'
 
+                - script: |
+                    echo $(COMMIT_SHA)
```

For consistency with the other one.
```suggestion
                    echo "commit: $(COMMIT_SHA)"
```